integration_tests/snaps/simple-svn/snapcraft.yaml

```diff
@@ -0,0 +1,11 @@
+name: test-package
+version: 0.1
+summary: svn integration test
+description: use svn
+confinement: strict
+
+parts:
+    svn:
+        plugin: make
+        source: .
+        source-type: svn
```

Comment:
This looks good, but it's missing that make file, and the test file. I'm not sure if it's work in progress, or you forgot to add the files to git.